GitCommands/Settings/AppSettings.cs

```diff
@@ -686,6 +686,12 @@ public static bool RebaseAutoStash
             set => SetBool("RebaseAutostash", value);
         }
 
+        public static bool SkipRebaseDialog
```

Comment:
The logic should be to show the dialog by default
The setting should be inverted:
AlwaysShowRebaseDialog

Reply:
Inversion dropped (see below), I still want to have the warning by default.